gui/configuration_provider.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+import os.path
+from idact.detail.environment.environment_impl import EnvironmentImpl
+from idact.detail.environment.environment_text_serialization import \
+    serialize_environment
+
+
+class ConfigurationProvider:
+    def __init__(self):
+        self.filename = ".idact.conf"
```

Comment:
I'm not sure if we need to create the `.idact.conf` file in gui. Is it really necessary? Doesn't idact do that itself? The files we definately want to create here are the `args.json` and `native_args.json` and that would also mean that we want them in the home directory, so using them would also change(currently they are inside gui). Maybe we need some function like save arguments in this class?

Reply:
What do you mean by asking if idact does it itself?
We create this file only if there is no '.idact.conf' in home directory - such situation can happen.